server/graphql/core/src/loader/document_registry.rs

```diff
@@ -6,6 +6,32 @@ use async_graphql::*;
 use service::service_provider::ServiceProvider;
 use std::collections::HashMap;
 
+pub struct DocumentRegistryLoader {
+    pub service_provider: Data<ServiceProvider>,
+}
+
+#[async_trait::async_trait]
+impl Loader<String> for DocumentRegistryLoader {
+    type Value = DocumentRegistry;
+    type Error = RepositoryError;
+
+    async fn load(&self, entries: &[String]) -> Result<HashMap<String, Self::Value>, Self::Error> {
```

Comment:
should this be document_types (vs entries)

Reply:
Thanks done. For filters I tend to use some generic name like `value` to help with copy and paste but this doesn't work very well for loaders. [9077397](https://github.com/openmsupply/open-msupply/pull/184/commits/907739759593a0a28e07adeebe21ca489a003e7c)